File: Semana 3/Tarea encargada de funciones (cuaderno)/python/Ejercicio10.py
def ejercicio10(base, exponente=2):
    resultado = 1.0
    for _ in range(exponente):
        resultado *= base
    return resultado

print(f"4 al cuadrado = {ejercicio10(4)}")
print(f"2 al cubo = {ejercicio10(2, 3)}")

File: Semana 3/Tarea encargada de funciones (cuaderno)/C++/Ejercicio10.cpp
#include <iostream>
using namespace std;

double Ejercicio10(double base, int exponente = 2) {
    double resultado = 1;
    for (int i = 0; i < exponente; i++) {
        resultado *= base;
    }
    return resultado;
}

int main() {
    cout << "4 al cuadrado = " << Ejercicio10(4) << endl;
    cout << "2 al cubo = " << Ejercicio10(2, 3) << endl;
    return 0;
}
